tests/integration/awslambda/test_lambda_common.py

```diff
@@ -173,3 +176,52 @@ def test_uncaught_exception_invoke(self, lambda_client, multiruntime_lambda, sna
         )
         assert "FunctionError" in invocation_result
         snapshot.match("error_result", invocation_result)
+
+    @pytest.mark.skip_snapshot_verify(
+        paths=[
+            "$..CodeSha256",  # works locally but unfortunately still produces a different hash in CI
+            "$..SnapStart",  # FIXME needs to be added to CreateFunction + all snapshots regenerated
+        ]
+    )
+    # this does only work on al2 lambdas, except provided.al2.
```

Comment:
:raised_eyebrow:

Reply:
Yes, its kinda weird this is only in al2, because its all in the bootstrap shell script anyway.